src/components/Form.js

```diff
@@ -0,0 +1,23 @@
+// import { useState } from "react";
+// import ErrorMessage from "./ErrorMessage";
+
+const Form = (setChosenDate) => {
+
+    return (
+        //Credit: got help figuring out how to hold a selected date in state from this tutorial: https://www.youtube.com/watch?v=dobnC4p5PYw
+        <>
+            <div className="form">
+                <form>
+                    <label htmlFor="date">Please  select the date you'd like to view:</label>
+                    <input className="form" type="date" id="date"
+                        placeholder="dd-mm-yyyy"
+                        min="2013-02-22"
+                        max="2023-12-31" onChange={e => setChosenDate(e.target.value)} />
```

Comment:
I am not sure if chosenDate is something you would like to set inside Form Component, you can now get hold of the user selected date like this! but if you want to move the date state over to App that works too!! ahhh out of time!nowwww

```js
const Form = ({setChosenDate}) => {
    
    const [date, setDate] = useState('');
    
    return (
        //Credit: got help figuring out how to hold a selected date in state from this tutorial: https://www.youtube.com/watch?v=dobnC4p5PYw
        <>
           <div className="form">
                <form>
                    <label htmlFor="date">Please select the date you'd like to view:</label>
                    <input
                    className="form"
                    type="date"
                    id="date"
                    placeholder="dd-mm-yyyy"
                    min="2013-02-22"
                    max="2023-12-31"
                    onChange={(e)=>setChosenDate(e.target.value)}
                    />
                </form>
                {date && <p>Selected Date: {date}</p>}
            </div>
        </>
    )
}
```